lib/business/connectivity_manager.dart

```diff
@@ -28,13 +30,28 @@ class ConnectivityManager {
   bool electrumConnected = true;
   bool nguConnected = false;
 
-  bool torTemporarilyDisabled = false;
+  DateTime? torTemporarilyDisabledTimeStamp;
 
   final StreamController<ConnectivityManagerEvent> events =
       StreamController.broadcast();
 
   static final ConnectivityManager _instance = ConnectivityManager._internal();
 
+  //Checks if the timeout has expired and if so, resets the temporary disablement
+  bool get torTemporarilyDisabled {
+    if (torTemporarilyDisabledTimeStamp != null &&
+        torTemporarilyDisabledTimeStamp!.isAfter(DateTime.now())) {
+      return true;
```

Comment:
It's opposite right? If we're after the timestamp tor is not disabled so we return false and clear the timestamp?

Reply:
In line number 51 we are setting the time to future. so the condition is correct we check against current time and if the current time is greater than ```torTemporarilyDisabledTimeStamp``` that means the disablement state is expired